File: app/src/main/java/com/byd/dashcast/proxy/ProxyDisplayResourceOwner.java
package com.byd.dashcast.proxy;

import android.os.Binder;
import android.os.IBinder;

/** Process-owned token for transient VirtualDisplays created by ProxyDaemon. */
final class ProxyDisplayResourceOwner {

    private static final IBinder TOKEN = new Binder();

    private ProxyDisplayResourceOwner() {}

    static IBinder token() {
        return TOKEN;
    }
}

File: app/src/main/java/com/byd/dashcast/proxy/ProxyDisplayResourceOwner.kt
package com.byd.dashcast.proxy

import android.os.Binder
import android.os.IBinder

/** Process-owned token for transient VirtualDisplays created by ProxyDaemon. */
internal object ProxyDisplayResourceOwner {

    private val TOKEN: IBinder = Binder()

    @JvmStatic
    fun token(): IBinder = TOKEN
}
